CONTRIBUTING.md

```diff
@@ -72,6 +72,16 @@ This is done by running `go generate` in the following way.
 $ go generate ./stdlib
 ```
 
+### Regenerating checksums
+
+If you modify any `.rust` files in the repository, checksums must be recomputed.
```

Comment:
This should be `.rs` files aka Rust files.

Reply:
Ah yes! Doh